Tests/PackageLoadingTests/PkgConfigParserTests.swift

```diff
@@ -105,6 +105,20 @@ final class PkgConfigParserTests: XCTestCase {
         }
     }
 
+    func testDummyDependency() {
+        try! loadPCFile("dummy_dependency.pc") { parser in
```

Comment:
make the function throwing and replace the `try!` with `try`

Reply:
This is copy-and-pasted from other test cases. Shall we `try` on every one?

I think it's okay to crash the test because such failure is unexpected and likely programming or environment error.

Reply:
Existing `try!` are likely from throwing tests weren't a thing. I don't think we have to necessarily update existing tests, but when writing new tests we should be avoiding `try!`.